fixtures/describe.html

```diff
@@ -11,5 +11,12 @@
       </li>
       <li>milk</li>
     </ul>
+    <script>
+      setTimeout(function () {
```

Comment:
Can you create a new file for the testing? Like `wait_elements.html`

Reply:
done.